Lokasenna_GUI v2/Library/Classes/Class - TextEditor.lua

```diff
@@ -294,7 +294,7 @@ function GUI.TextEditor:ontype(char, mod)
 		end
 
 	-- Typeable chars
-	elseif GUI.clamp(32, char, 254) == char then
+	elseif not disableEditing and GUI.clamp(32, char, 254) == char then
```

Comment:
This should be self.disableEditing

Reply:
good catch, i probably should have copypasted from the vendor version in the other repo